api/lua/definition/poptracker.lua

```diff
@@ -0,0 +1,548 @@
+---@param variableName string
+---@return any value of variable or nil
+function AutoTracker:ReadVariable(variableName) end
+
+---@alias AutoTrackingBackendName "SNES" | "UAT" | "AP"
+
+---@alias AutoTrackingState
+---| -1 # Unavailable
+---| 0 # Disabled
+---| 1 # Disconnected
+---| 2 # Socket Connected (not ready)
+---| 3 # Game/Consolve connected (ready)
+
+---get an integer corresponding to the current state of an auto-tracking backend.
+-- Available since 0.20.2.
+---@param backendName AutoTrackingBackendName name of the backend (case sensitive)
+---@return AutoTrackingState state of the backend
+function AutoTracker:GetConnectionState(backendName) end
+
+
+---- Segment ---
+
+---NOTE: the segment API is provided by `AutoTracker`, but this is not backwards compatible.
+-- `ReadU8` should only be used on global AutoTracker and `ReadUInt8` on segment from watch callback.
+---@class Segment
+Segment = {}
+
+---read 8bit unsigned integer, may return 0 if used on AutoTracker instead of Segment
+---@param address integer (absolute) address to read
+---@return integer value at address
+function Segment:ReadUInt8(address) end
+
+---read 16bit unsigned integer, may return 0 if used on AutoTracker instead of Segment
+---@param address integer (absolute) address to read
+---@return integer value at address
+function Segment:ReadUInt16(address) end
+
+---read 24bit unsigned integer, may return 0 if used on AutoTracker instead of Segment
+---@param address integer (absolute) address to read
+---@return integer value at address
+function Segment:ReadUInt24(address) end
+
+---read 32bit unsigned integer, may return 0 if used on AutoTracker instead of Segment
+---@param address integer (absolute) address to read
+---@return integer value at address
+function Segment:ReadUInt32(address) end
+
+
+---- VariableStore ----
+
+---NOTE: the store API is provided by `AutoTracker`, but this may change in the future.
+---@class VariableStore
+VariableStore = {}
+
+---Returns what the remote (UAT) sent for the variableName.
+---@param variableName string
+---@return any value of variable or nil
+function VariableStore:ReadVariable(variableName) end
+
+
+---- Archipelago ----
+
+---@class Archipelago
+Archipelago = {}
+
+---The slot number of the connected player or -1 if not connected
+---@type integer
+Archipelago.PlayerNumber = -1
+
+---The team number of the connected player, -1 if not connected or `nil` if unsupported.
+---Supported since 0.25.2.
+---@type integer
+Archipelago.TeamNumber = -1
+
+---Array of already checked location IDs or `nil` if unsupported.
+---Supported since 0.25.2.
+---@type integer[]
+Archipelago.CheckedLocations = {}
+
+---Array of unchecked/missing location IDs or `nil` if unsupported.
+---Supported since 0.25.2.
+---@type integer[]
+Archipelago.MissingLocations = {}
+
+---Add callback to be called when connecting to a (new) server and state should be cleared.
+---@param name string Identifier/name of this handler (for debugging)
```

Comment:
```suggestion
---@param name string identifier/name of this handler (for debugging)
```

similar problem with inconsistent lowercase/uppercase as with the start of comments we have for some param descriptions